bundles/binding/org.openhab.binding.maxcube/src/main/java/org/openhab/binding/maxcube/internal/message/MessageType.java

```diff
@@ -15,5 +15,19 @@
 * @since 1.4.0
 */
 public enum MessageType {
-	H, M, C, L, S
+	H("H:"),
+	M("M:"),
+	C("C:"), 
+	L("L:"),
+	S("S:");
+	
+	private String messageIndicator;
```

Comment:
I know what mean, but I do not see a suitable way to get rid of this code.
I neet the message type before I compile the actual message object. So I can not use message.getMessage at this point in time.